e2e/03-intraledger-flow.e2e.spec.ts

```diff
@@ -18,11 +18,17 @@ describe("Validate Username Flow", () => {
   })
 
   it("Paste Username", async () => {
-    const usernameInput = await $(selector(LL.SendBitcoinScreen.input(), "Other", "[1]"))
-    await usernameInput.waitForDisplayed({ timeout })
-    await usernameInput.click()
-    await usernameInput.setValue(username)
-    await enter(usernameInput)
+    try {
```

Comment:
try catch in a test is an anti-pattern.

Reply:
you can read/add https://www.npmjs.com/package/eslint-plugin-jest for understanding test best practices